### ISSUE TITLE
TLS handshake error while using self-issued certificate, accessing from iOS devices (Chrome)

### ISSUE THREAD

**Issue by alexsannikov** [AdguardTeam/AdGuardHome#1101]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  *  0.99
* **How did you setup DNS configuration:**
  *  Router
* **If it's a router or IoT, please write device model:**
  *  RPI 3B+
* **Operating system and version:**
  *  Raspbian Buster

### Expected Behavior

No messages in log like:
2019/10/20 17:12:56 http: TLS handshake error from 192.168.7.202:49524: remote error: tls: unknown certificate
2019/10/20 17:13:18 http: TLS handshake error from 192.168.7.202:49528: remote error: tls: unknown certificate
2019/10/20 17:13:22 http: TLS handshake error from 192.168.7.202:49530: remote error: tls: unknown certificate
### Actual Behavior

I generated root certificate for self-signed CA. 
Based on rootCA certificate, I issued wild-card certificate (which includes several domain names and set of IP addresses in SAN field) and installed it to AdGuard home. I use this certificate also widely for my several home devices.
1. I installed rootCA certificate to MacOS 10.14 and set to trust it. When I open web-interface of AdGuard Home no any error messages in log;
2. I've sent the same rootCA certificate to my iOS 13.1.3 device, installed it into profile and st to trust this certificate. When I open web-interface of AdGuard Home from iOS Chrome I see error messages above anyway.
I accepted this certificate and confirmed to trust it in Safari, so no messages/errors when I come from this browser.
### Screenshots

<details><summary>Screenshot:</summary>

</details>

### Additional Information

Everything described [here](https://www.thesslstore.com/blog/trust-manually-installed-root-certificates-in-ios/) is done.

**Comment by ameshkov** [comment]
From what I read it sounds like an iOS issue, I don't fully understand what can we do from our side to change this behavior.

**Comment by alexsannikov** [comment]
I understand. Maybe, someone can confirm this behaviour. Anyway, I don't also think this is an AdGuard Home issue, but AGH is only software which complains about this in logs :).
Also, maybe I missed something and somebody can give my additional instructions or direction.
Actually, it looks like Chrome trust system Certificate storage, but something is happens wrongly during TLS phase between Chrome and AGH. 
If you think this is not an issue, you can close this ticket.
Thanks!

**Comment by ameshkov** [comment]
> Anyway, I don't also think this is an AdGuard Home issue, but AGH is only software which complains about this in logs

Well, others can also do this. When the browser encounters a certificate it does not trust, it sends a TLS alert "unknown_ca" to the server right before closing the connection. I guess that's what's being logged.

**Comment by alexsannikov** [comment]
Ok, looks like my bad, I used old certificate.
I changed settings to use path to new SSL certificate and problem has gone.
Sorry for inconvenience.

SUMMARY:
Problem: I had messages "TLS handshake error while using self-issued certificate" while used self-signed SSL certificate.
Root cause: I used certificate issued with some errors (cannot check which exactly)
Solution: I started to use correct certificate and don't see error messages for now.

Thanks!
